nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ListFile.java

```diff
@@ -280,6 +280,8 @@
     public static final String FILE_SIZE_ATTRIBUTE = "file.size";
     public static final String FILE_OWNER_ATTRIBUTE = "file.owner";
     public static final String FILE_GROUP_ATTRIBUTE = "file.group";
+    public static final String FILE_LIST_BATCH_COUNT = "file.list.batch.count";
```

Comment:
I think we also need to have a description of these attributes. 
        @WritesAttribute(attribute=ListFile.FILE_LIST_BATCH_COUNT , description="Batch Count...."),